pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -103,6 +106,23 @@ func init() {
 			eventsRule(),
 		},
 	})
+
+	// serviceaccount-controller
+	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + InfraServiceAccountControllerServiceAccountName},
+		Rules: []rbac.PolicyRule{
+			rbac.NewRule(ReadWrite...).Groups(kapiGroup).Resources("serviceaccounts").RuleOrDie(),
+			eventsRule(),
+		},
+	})
+
+	// serviceaccount-tokens-controller
+	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + InfraServiceAccountTokensControllerServiceAccountName},
```

Comment:
@deads2k wonder if we want/need this... this controller will use the RootClientBuilder which means privileged client will be used.